Firebase Storage

CallMeMaverick opened the pull request:
# Description
Implemented functionality for storing images in Firebase storage. You can log in to Firebase by using a shared gmail account we used for Mongo. 
The endpoint was tested using Postman and mocked users.

**_Please make sure that if you're testing locally, you must:_**
1. Add a JSON file that you can find in Google Spreadsheets
2. Update `.env` to include `BUCKET_URL_FIREBASE`
3. Overcome authentication middleware by commenting out the implementation 

Closes #84

CallMeMaverick committed:
config(gitignore): update .gitignore to include firebase service account file

CallMeMaverick committed:
config(dependencies): install firebase-admin

CallMeMaverick committed:
config(firebase): firebase initialization

Implemented initializeFirebase function to initialize the app

CallMeMaverick committed:
feat(image-service): implement image service

CallMeMaverick committed:
config(firebase): BUCKET_URL_FIREBASE

CallMeMaverick committed:
feat(image-controller): implement image controller

CallMeMaverick committed:
feat(server-setup): initialize firebase before the app

CallMeMaverick committed:
feat(user-router): add path for adding an image

CallMeMaverick committed:
chore: minor clean-up

CallMeMaverick committed:
docs(swagger): document user router

Co-authored-by: Yaroslav Yakovlev <108124402+Yaroslav-Yakovlev@users.noreply.github.com>

nrevutska reviewed (changes requested):
we don't need separate controller for picture, use userController to operate with pictures for user

CallMeMaverick committed:
refactor(image-controller): remove image controller

Since image is not separate entity, delete its controller and move handling logic to user controller, to which is belongs

CallMeMaverick committed:
feat(user-router): update user router

Updated user router to include uploading an image
